gnovm/pkg/repl/repl_test.go

```diff
@@ -0,0 +1,110 @@
+package repl
+
+import (
+	"testing"
+
+	"github.com/jaekwon/testify/require"
+)
+
+type step struct {
+	Line   string
+	Error  string
+	Result string
+}
+
+var fixtures = []struct {
+	Name      string
+	CodeSteps []step
+}{
```

Comment:
Nitpick: Why not move this into `TestRepl`, so it's not available package-wide?

Reply:
I make this just for convenience, to make it easier to read the tests themselves. I can move them inside the test, or I can change the package name to repl_test, making it only available on tests. WDYT?

Reply:
Do you foresee this structure being used outside of this one test? I'd move it to the test for simplicity, because it can't really be shared between tests as it's a var declaration